azerothcore/azerothcore-wotlk
# feat(Core/Battleground): Config to allow/disallow ability to Share Quest & Ready Check

## CHANGES PROPOSED:
-  Add a config to make it possible to disallow Sharing Quests in a Battleground
-  Add a config to make it possible to disallow Ready Check in a Battleground


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1577



## TESTS PERFORMED:
- Builds
- Checked in-game (works!)



## HOW TO TEST THE CHANGES:

1. Pull the PR
2. Make sure you change your config file to the updated one
3. Log 2 chars and go into a group
4. .debug bg
5. queue as a group to BG
6. Try to share quest/run ready check
7. Redo the tests but after you change the config to 1/0
## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
